examples/lights/lights.c

```diff
@@ -1,60 +1,110 @@
-/**
- * Copyright (C) 2009 Ubixum, Inc. 
- * Copyright (C) 2016 Matthew Iselin <miselin@google.com>
- * Copyright (C) 2017 Kyle Robbertze <krobbertze@gmail.com>
- *
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 2.1 of the License, or (at your option) any later version.
- *
- * This library is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
+// Copyright (C) 2009 Ubixum, Inc.
+// Copyright (C) 2016 Matthew Iselin <miselin@google.com>
+// Copyright (C) 2017 Kyle Robbertze <krobbertze@gmail.com>
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+/** \file lights.c
+ * Example firmware that flashes LEDs.
  *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this library; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
- **/
+ * On the Atlys this is LD14 (DONE), on the FX2 miniboard D1 and D2 and on the 
+ * Opsis it is D2
+ */
 
 #include <fx2regs.h>
+#include <fx2types.h>
 #include <delay.h>
 
 void init_lights(void) {
     // Set pins to port mode instead of GPIF master/slave mode
     IFCONFIG &= ~(bmIFCFGMASK);
+    // No alternate functions
```

Comment:
The port the Atlys uses has no alternative functions, thus there is no config register to set/change